test/testcafe/spec/UnlockAccountChallengeIdentifierFirst_spec.js

```diff
@@ -106,6 +107,8 @@ const identifyLockedUserMockWithOneAuthenticator = RequestMock()
   .respond(xhrIdentifyWithUnlock)
   .onRequestTo('http://localhost:3000/idp/idx/unlock-account')
   .respond(xhrUserUnlockIdentifierFirst)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
```

Comment:
Missing request mock.
Test was working before because it used mock server from playground, see DISABLE_MOCK_SERVER